iset.mm

```diff
@@ -58497,6 +58497,20 @@ ordinals implies excluded middle ( ~ ordsucunielexmid ).  (Contributed
       MUNUOUP $.
   $}
 
+  ${
+    $d A x $.  $d B x $.
+    $( If we remove a single element from ` _om ` then put it back in, we end
+       up with ` _om ` back.  This strengthens ~ difsnss from subset to
+       equality but the proof relies on equality being decidable.  (Contributed
+       by Jim Kingdon, 11-Jun-2022.) $)
+    omdifsnid $p |- ( B e. _om -> ( ( _om \ { B } ) u. { B } ) = _om ) $=
```

Comment:
This generalizes, to `( ( A \ B ) u. B ) = A` when `x e. A -> DECID x e. B`

Reply:
Hmm, interesting.I too noticed that it could be more general although I don't remember exactly which generalization I was thinking of (probably something similar to the above but not identical).

It is fair to say that I have `_om` on the brain which is partly why I didn't go right to the generalizations. The reason for this fixation is the question posed at https://github.com/metamath/set.mm/issues/2595 - if a set is countable and infinite can we conclude it is countably infinite? (that is, `_om ~<_ A /\ A ~<_ _om -> A ~~ _om` ).

Reply:
The above-suggested generalization and one other (which I think is the one I had in mind before) is now an issue at https://github.com/metamath/set.mm/issues/2618